fpmsyncd/fpmlink.cpp

```diff
@@ -71,7 +71,7 @@ FpmLink::~FpmLink()
 void FpmLink::accept()
 {
     struct sockaddr_in client_addr;
-    socklen_t client_len;
+    socklen_t client_len = 0;
```

Comment:
in man 3 accept
The addrlen argument is a value-result argument: the caller must initialize it to contain the size (in bytes) of  the  structure pointed to by addr; on return it will contain the actual size of the peer address.

I agree previous code is broken, but it seems we shoud set client_len = sizeof(struct sockaddr_in)

Reply:
Agree. Will it be better with "client_len = sizeof(client_addr)"?